### ISSUE TITLE
Spurious conflicted copies due to zero'ed updated_at_timestamp

### ISSUE THREAD

**Issue by PeterCxy** [standardnotes/web#982]
My self-hosted Standard Notes web app (along with a self-hosted backend server) is creating a lot of spurious conflicted copies, similar to what is described in https://github.com/standardnotes/forum/issues/928. After a bit of digging, it seems to me that the updated_at_timestamp is sometimes zeroed out in the sync requests (at least in my case, on Firefox 99, on Linux). Attached is a screenshot of the request content when this happened.

![2022-04-15T14:21:58,502617772-04:00](https://user-images.githubusercontent.com/4532423/163607513-500e5720-533a-4835-a964-4a1ba2b259d4.png)

**Comment by moughxyz** [comment]
Hmm, can't replicate this. Although this is interesting:

> Eventually, the status in the upper-right corner will be stuck at "Saving..."

Is this correlated with your server health at all? Would seem like it would be stuck at saving if server is not responding properly.

**Comment by PeterCxy** [comment]
@moughxyz No, pretty sure the server is running perfectly fine. The F12 network request debugger shows no outstanding request when this happens.

And also, it doesn't happen before 3a2ff2f.

**Comment by moughxyz** [comment]
Hmm, would definitely love to be able to reproduce this, as these new changes will be going to prod soon and they will become inescapable for both of us, if indeed it is a general issue 🙂 I presume for step 1 of your replication steps, you created a new note, and not edited an existing note?

Based on the timestamps it would seem like you created a new note, typed really fast, and never got a successful sync request through (otherwise the timestamps would have been updated after the first save).

**Comment by PeterCxy** [comment]
> I presume for step 1 of your replication steps, you created a new note, and not edited an existing note?

No, it happens for both existing and new notes, including those that already have had a successful sync. That's why it's so weird that the timestamps got somehow reset to 0.

**Comment by PeterCxy** [comment]
@moughxyz I did some more testing and it seems that I couldn't reproduce with the plain editor either. However, when I switch to any one of the Markdown editor, I can trigger this pretty reliably -- start typing, and stop for it to trigger sync, and start typing again before the sync finishes. Repeat a few times and this will start to happen.

**Comment by moughxyz** [comment]
Ah indeed, I am seeing some weird behavior with editors. Thanks for the tip! Will report back after I complete a deeper investigation.
